python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -328,7 +364,18 @@ def get_params(self) -> Iterator[PipesParams]:
         """
 
     @abstractmethod
-    def download_messages_chunk(self, index: int, params: PipesParams) -> Optional[str]: ...
+    def download_messages_parts(
```

Comment:
not sure what's a good new name for this method

Reply:
How about just `download_messages`